app/utils/transformers/AdditionalReferencesTransformer.scala

```diff
@@ -57,7 +60,7 @@ class AdditionalReferencesTransformer @Inject() (referenceDataConnector: Referen
     genericTransform(additionalReferences)(_.typeValue) {
       case (TempAdditionalReference(underlying, typeValue), index) =>
         setSequenceNumber(AdditionalReferenceSection(hcIndex, itemIndex, index), underlying.sequenceNumber) andThen
-          set(AdditionalReferencePage(hcIndex, itemIndex, index), typeValue) andThen
+          set(AdditionalReferenceTypeItemPage(hcIndex, itemIndex, index), typeValue) andThen
```

Comment:
Alternatively, instead of an alias, you can add the relevant imports inside the method